targets/nuttx-stm32f4/README.md

```diff
@@ -16,6 +16,8 @@ git clone https://github.com/jerryscript-project/jerryscript.git
 git clone https://bitbucket.org/nuttx/nuttx.git
 git clone https://bitbucket.org/nuttx/apps.git
 git clone https://github.com/texane/stlink.git
+cd stlink
+git checkout 3d24377c2878fc328fb09b67cdfbda529d209417
```

Comment:
I'm not sure I agree with this. Why do we have to fix stlink to a given hash? In the commit message you've mentioned that this saved us from the need for frequent updates to the readme. But what is changing so frequently in stlink that triggers updates here?

If we specify the hash here, we will have a "last known good version", for sure. But we lose any potential benefit from the development that happens on stlink master. And if we wanted to follow up on that then we had to update readme with new "last known good version" hashes over and over again.